### ISSUE TITLE
website figures

### ISSUE THREAD

**Issue by rb643** [autism-research-centre/Autism-Gradients#14]
Owen would you be able to give some guidance on how figures should be embedded in the notebook markdown files?

**Comment by owenparsons** [comment]
Yep, you just need to put them somewhere in the repository that isn't in the _posts folder, and then you can adjust the img tag in the markdown file. You can put it in img/post_imgs directory that I set up.

**Comment by rb643** [comment]
Sorry for the ignorance but what formatting do you need for markdown? ![png](link) doesn't seem to work :)

**Comment by owenparsons** [comment]
It needs to be in the format  ![Alt Text](url)

For url, you could put the full url or I think it should start from the base directory, so  ![Alt Text](/img/post_imgs/image_name.png) should work

**Comment by owenparsons** [comment]
Ahhh, just realised it's trying to render the markdown. I'll send that message again on slack

**Comment by owenparsons** [comment]
![Figure 3](https://autism-research-centre.github.io/Autism-Gradients/img/post_imgs/output_3_0.png) should work or ![Figure 3](/img/post_imgs/output_3_0.png)
